code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -255,7 +255,8 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 
 			if(bitf == MARINE_CAN_BUY_UNIFORM && ishumanbasic(usr))
 				var/mob/living/carbon/human/H = usr
-				new /obj/item/radio/headset/mainship/marine(loc, H.assigned_squad, vendor_role)
+				var/headset_type = H.faction == FACTION_TERRAGOV ? /obj/item/radio/headset/mainship/marine : /obj/item/radio/headset/mainship/marine/rebel
```

Comment:
i believe it would be better to refactor how headset works, otherwise just pillage someone headset to get their comms, make maybe channels based on your role or something ?

Reply:
this is fine for now, i'll consider a refactor later

Reply:
this is fine for now, i'll consider a refactor later